environments/icds-cas/postgresql.yml

```diff
@@ -37,6 +37,7 @@ pgbouncer_override:
   pgbouncer_reserve_pool: 5
 
 postgres_override:
+  postgresql_max_standby_delay: 180s
```

Comment:
This should be int ?

Reply:
updated https://github.com/dimagi/commcare-cloud/pull/3090/commits/3f3989c3679b1bab925a0aabed9ab2873c10f953